static/js/admin.js

```diff
@@ -383,95 +372,98 @@ WebcomAdmin.sliderMetaBoxes = function($) {
     $('#ss_slides_all').sortable({
       handle : 'h3.hndle',
       placeholder : 'sortable-placeholder',
-      sort : function() {
-        $('.sortable-placeholder').height( $(this).find('.ui-sortable-helper').height() );
+      sort : function () {
+        $('.sortable-placeholder').height($(this).find('.ui-sortable-helper').height());
       },
-      stop : function ( event, ui ) {
-        var html5iframe = $(ui.item).find('iframe.wysihtml5-sandbox'),
-            toolbar = $(ui.item).find('div[id^="wysihtml5-toolbar["]'),
-            textarea = $(ui.item).find('textarea[id^="ss_slide_caption["]');
+      stop : function (event, ui) {
+        const html5iframe = $(ui.item).find('iframe.wysihtml5-sandbox'),
+          toolbar = $(ui.item).find('div[id^="wysihtml5-toolbar["]'),
+          textarea = $(ui.item).find('textarea[id^="ss_slide_caption["]');
 
         textarea.show();
         html5iframe.remove();
         toolbar.hide();
 
-        var editor = new wysihtml5.Editor(textarea.attr('id'), { // id of textarea element
-            toolbar: toolbar.attr('id'), // id of toolbar element
-            stylesheets: [THEME_CSS_URL + "editor.css"],
-            parserRules: wysihtml5ParserRules, // defined in parser rules set
+        // eslint-disable-next-line no-new
+        new wysihtml5.Editor(textarea.attr('id'), { // id of textarea element
+          toolbar: toolbar.attr('id'), // id of toolbar element
+          stylesheets: [`${THEME_CSS_URL}editor.css`],
+          parserRules: wysihtml5ParserRules // defined in parser rules set
         });
       },
-      update : function( event, ui ) {
-          updateSliderSortOrder();
+      update : function () {
+        updateSliderSortOrder();
       },
       tolerance :'pointer'
     });
 
 
     // Toggle slide with header click
-    $('#slider_slides').delegate('.custom_repeatable .hndle', 'click', function() {
-      $(this).siblings('.inside').toggle().end().parent().toggleClass('closed');
+    $('#slider_slides').delegate('.custom_repeatable .hndle', 'click', function () {
```

Comment:
`.delegate()` was deprecated in jQuery 3.x

Reply:
Fixed